spec/Cabin/Infotainment.vspec

```diff
@@ -188,7 +188,36 @@ HMI.DayNightMode:
   type: actuator
   allowed: ['DAY', 'NIGHT']
   description: Current display theme
-  
+
+# The hypothetical examples below could possibly be handled in an overlay, if we do not see that it is generally usable
+
+# Hypothetical example - adding another power optimize level within "Cabin"
+
+#include PowerOptimize/PowerOptimizeBranch.vspec
+
+# Hypothetical example - adding supporting signals
```

Comment:
Trying to link how the structure could be aligned with the proposal in https://github.com/COVESA/vehicle_signal_specification/pull/520.

( The question is off course if we consider it general enough to add another brightness signal like this to standard catalog, or if rather shall be a project/platform-specific overlay)